web/src/main/webapp/initialObject.ts

```diff
@@ -48,7 +53,7 @@ export interface Merged {
     second: DataLoaded;
 }
 
-export type DataLoaded = FilesLoaded | TablesLoaded | HillviewLogs | IDatasetSerialization | Merged;
+export type DataLoaded = FilesLoaded | TablesLoaded | DeltaTableLoaded | HillviewLogs | IDatasetSerialization | Merged;
```

Comment:
Do you need here a new type or can you just reuse FilesLoaded?

Reply:
I think we should have a new type due to the following reasons:
1. `FilesLoaded` needs a `FileSetDescription`, current that is generated at the back end and not returned to the front end.
2. Having a separate `DeltaTableLoaded` interface allows the front end to update the UI specifically for delta table. For example, current we can set the page title as "loaded from delta table" rather than "loaded from files", otherwise it can be confusing for the user. In the future we can also add a drop down menu for user to choose snapshot version once a delta table is loaded.